packages/common/src/util.js

```diff
@@ -687,13 +687,18 @@ util.getUserAgentFromPackageJson = getUserAgentFromPackageJson;
  * Wraps a callback style function to conditionally return a promise.
  *
  * @param {function} originalMethod - The method to promisify.
+ * @param {object=} options - Promise options.
+ * @param {boolean} options.singular - Resolve the promise with single arg
```

Comment:
Well, I think the closest would be spread, but that would imply that we're passing multiple arguments which this does not. This is unique behavior for us since we initially decided to only ever pass back arrays.